canonical-web-and-design/mir-server.io
# Convert to flask site

## Done
Convert eleventy site into flask site. Fixes https://github.com/canonical-web-and-design/web-squad/issues/5217

## QA
- Go to https://mir-server-io-170.demos.haus/ check links, navigation work, no broken images
- Check global navigation is showing
- Check console is not giving any js errors
- Check title, metadata, description looks correct
- Check anything you can think of against https://mir-server.io/
- Checkout this branch and run locally with `dotrun`
- Checkout this branch and run locally with `./run`. You may need to `chmod +x entrypoint` if you do this the first time
- Test 404 page, e.g. https://mir-server-io-170.demos.haus/not-found-page
- Test sitemap https://mir-server-io-170.demos.haus/sitemap.xml

Other checks that are taken care of by Github actions:
- Run-image should pass
- Tests should pass (test_routes.py is reviewed by this action)
- Python Linting should pass
